language/tools/move-mv-llvm-compiler/src/stackless/rttydesc.rs

```diff
@@ -120,6 +124,10 @@ fn type_name(mty: &mty::Type) -> String {
     use mty::{PrimitiveType, Type};
     let name = match mty {
         Type::Primitive(PrimitiveType::U64) => "u64",
+        Type::Vector(elt_ty) => match **elt_ty {
+            Type::Primitive(PrimitiveType::U64) => "vector<u64>",
+            _ => todo!(),
+        },
```

Comment:
There seems to be a formatter for `Type` in `ty.rs`. So instead of writing this from scratch, can we just replace the whole function with something like:
`mty.display(&my_module_environment.env.get_type_display_ctx())`
where you'll just need to provide `my_module_environment`. It looks like your outputting exactly the same strings as that formatter, and their code already handles every case.

Reply:
Following myself up-- I verified locally that the `Type` formatter does function as expected. Likewise for the similar functions below.

Reply:
I have made this change.